geometry_msgs/Twist command
float64[] joint_velocity_measured
float64[] joint_velocity_commanded
float64[] joint_velocity_error
float64[] joint_effort_measured
float64[] joint_effort_commanded
float64[] joint_effort_error
string[] joint_names

